region/crateria/west.json

```diff
@@ -1274,7 +1401,8 @@
                       "type": "PowerBomb",
                       "count": 3
                     }}
-                  ]
+                  ],
+                  "obstacles": [{"id": "A", "requires": []}]
```

Comment:
It technically needs only 2 PBs to break enough of the blocks to use the runway. This would likely require another 1-2 nodes or 2 more obstacles. You can simply add a fixme for now.

Reply:
I'm thinking the 2 more obstacles method would be best because then you can correctly model opening the door before sparking.